### ISSUE TITLE
Loss:Nan [Semantic Segmentation Experiment on Synthia]

### ISSUE THREAD

**Issue by ihanwen99** [xingyul/meteornet#6]
I am new one in tensorflow, I have faced the problem loss: nan in Semantic Segmentation Experiment on Synthia.

I downloaded the generated processed dataset, and `python gen_label_weights` as the README says. While in this step, I faced a problem 'invalid value encountered in true_divide', so I change the code with `np.stderr(divide='ignore', invalid='ignore')`. Then it worked successfully.

While when I train with dataset, because of my GPU memory is not enough for batch_size=12, I change it to 1/2/3/4/5, it face another promblem that **loss:nan.** Is there any mistake I make?

Thanks greatly.

**Comment by ihanwen99** [comment]
It seems my mistakes, I reconfirm the path of dataset. 

Now it runs successfully. Thanks a lot!
